docs/open_source/integrate_tests/index.md

```diff
@@ -8,23 +8,14 @@ Once you've created a test suite for a model, integrating it into existing workf
 :hidden:
 
 ../../integrations/cicd/index
-../../integrations/cicd/pipeline.ipynb
```

Comment:
This should be here, and cicd/index should not

Reply:
to be conform with the others, it should be cicd/index in both the toc and the cell. I didn't like the idea of pointing to a notebook from here.